### DISCUSSION TITLE
Akka.NET 2023 Community Standups

### DISCUSSION THREAD

**Discussion by Aaronontheweb** [akkadotnet/akka.net#6350] [Community Standups]
Last year we were very successful in re-introducing our [Akka.NET Community Standups](https://github.com/akkadotnet/akka.net/discussions/5691) and we are going to continue that practice again in 2023! This is our thread for coordinating on upcoming standups.

Akka.NET Community Standups are a once-a-month meeting that happens on the second Wednesday of each month and is live-streamed via [Petabridge’s YouTube channel](https://www.youtube.com/c/PetabridgeAcademy).

# Structure of Each Standup

1. **Recap of most recent releases (10 min)** - we should talk about the work that was performed since the last standup, its impact on end-users, and what we learned in the course of performing that work.
2. **Plans for future work (10 min)** - what are we working on next, why, and why should users care?
3. **Opportunities to contribute (5 min)** - what help would we like from the Akka.NET end-user community? 
4. **[Show and Tell](https://github.com/akkadotnet/akka.net/discussions/5690) (~20 min)** - give members of the Akka.NET team and community opportunities to showcase their work built on top of Akka.NET. See https://github.com/akkadotnet/akka.net/discussions/5690 for opportunities to apply.
5. **Q&A (~10 min)** - show and tell presenters along with the [Akka.NET](http://Akka.NET) team should take questions via YouTube Live from the audience.

**Standups are second Wednesday of each month starting at 12pm CST.**

**Comment by Aaronontheweb** [comment]
Akka.NET January 18th, 2023 Community Standup agenda is live here: https://petabridge.notion.site/01-18-2023-Akka-NET-Community-Standup-7714d55750c643e5866404c9073ce238

Video stream / recording: https://www.youtube.com/watch?v=O0hYqjLP9Lc

**Comment by Aaronontheweb** [comment]
Akka.NET February 8th, 2023 Community Standup agenda live here: https://petabridge.notion.site/02-08-2023-Akka-NET-Community-Standup-cfb705a4b5cd44deaf447ec8fea30525

Video stream / recording: https://www.youtube.com/watch?v=f_edv4zoRlo

**Comment by Aaronontheweb** [comment]
Akka.NET April 12th, 2023 Community Standup agenda live here: https://petabridge.notion.site/04-12-2023-Akka-NET-Community-Standup-76fa2ffb6aff490ca1095e6ffc586804

Video stream / recording: https://www.youtube.com/watch?v=gLVRudC3ZQ0

**Comment by Aaronontheweb** [comment]
June 14th, 2023 Community Standup agenda live here: https://petabridge.notion.site/06-14-2023-Akka-NET-Community-Standup-2dcab2bf55b345cba598f52ebd39100e?pvs=25

Video stream / recording: https://www.youtube.com/watch?v=f-2j2UCHBTo

**Comment by Aaronontheweb** [comment]
August 09 community standup agenda is live here: https://petabridge.notion.site/08-09-2023-Akka-NET-Community-Standup-410f6f2ed7584b64a70e22fdc5709ffe?pvs=4

Video stream / recording: https://www.youtube.com/watch?v=ui12OQd_czI

**Comment by Aaronontheweb** [comment]
September 13th community standup agenda is live here: https://petabridge.notion.site/09-13-2023-Akka-NET-Community-Standup-fbc6a22461814fcaa6e7b5679261b94d?pvs=4

Video stream / recording: https://www.youtube.com/watch?v=WZ8J8FZGkkE

**Comment by Aaronontheweb** [comment]
October 13th community standup agenda is live here: https://petabridge.notion.site/10-13-2023-Akka-NET-Community-Standup-dcd8e61e53684a20904f825b364fe4ea?pvs=4

Video stream / recording: https://www.youtube.com/watch?v=5Ii35B3fSGE

**Comment by shelbyfranklin** [comment]
February 14th community standup agenda is live here: https://petabridge.notion.site/2-14-2024-Akka-NET-Community-Standup-a650163aaa64461aa331a6d972b024a9

Video stream / recording: https://youtube.com/live/NqygB68FwA4
